config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  root to: 'pages#home'
+  match 'welcome' => 'pages#home', via: 'get'#home'
```

Comment:
Please change from `#match` to `#get` the `via` option is not necessary in this case.